### DISCUSSION TITLE
Rotable markers or images

### DISCUSSION THREAD

**Discussion by cubecleveland** [geoman-io/leaflet-geoman#1092] [Questions & Help]
Hi, is there any way to achive  rotation on markers? use case is that we want to put an image Overlay or a marker on the map that can be rotated. use case is we are trying to achieve is to put images on the map that can be roatatable ..

**Comment by Falke-Design** [comment]
I use in my project [Leaflet.RotatedMarker](https://github.com/bbecquet/Leaflet.RotatedMarker) but it doesn't work with shadow. A gui interface is missing, so you need to create one by your self.

**Reply by cubecleveland** [reply]
Is there any way to connect it to the current Geoman plugin, or do we need to build the entire thing from scratch ? putting the markers for rotating etc... ?
